pyiode/iode_database/variables_database.pyx

```diff
@@ -936,6 +949,7 @@ cdef class Variables(_AbstractDatabase):
             filepath to the input file.
         eps_test: float
             ???
+        # GDM> is there a reason this is called series instead of variables
```

Comment:
Arguments names for the seasonal_adjustment() method are taken from the French documentation of the `WsSeasonAdj` IODE command.

The documentation of seasonal_adjustment() has been rewritten in the PR #572